test/controllers/renewal/CustomersOutsideIsUKControllerSpec.scala

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.renewal
+
+import connectors.DataCacheConnector
+import models.Country
+import models.businessmatching._
+import models.renewal._
+import org.jsoup.Jsoup
+import org.mockito.Matchers._
+import org.mockito.Mockito._
+import play.api.i18n.Messages
+import play.api.inject._
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.mvc.{AnyContentAsFormUrlEncoded, Result}
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import services.RenewalService
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.cache.client.CacheMap
+import uk.gov.hmrc.play.frontend.auth.AuthContext
+import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
+import utils.{AmlsSpec, AuthorisedFixture}
+
+import scala.concurrent.Future
+
+class CustomersOutsideIsUKControllerSpec extends AmlsSpec {
+
+  trait Fixture extends AuthorisedFixture {
+    self =>
+    val request = addToken(authRequest)
+
+    implicit val authContext = mock[AuthContext]
+    implicit val headerCarrier = HeaderCarrier()
+
```

Comment:
both implicits should already be available by extending AmlsSpec

Reply:
Done